### ISSUE TITLE
`isGeolocationEnabled` returns true before user has given permission

### ISSUE THREAD

**Issue by tyleraadams** [no23reason/react-geolocated#215]
This is occurring when setting `suppressLocationOnMount` to `true`, but even when that is `false`, `isGeolocationEnabled` is returning `true` initially before changing to `false`. 

I would expect that this value should be `false` by default until user gives permission to use location.

here are the settings i am using to initialize:
```
export default geolocated({
  suppressLocationOnMount: true,
  userDecisionTimeout: 5000
})(GetUserLocation);
```

[![Edit clever-sun-1emne](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/clever-sun-1emne?fontsize=14)

Also wanted to give kudos for a great library 👍!

**Comment by tyleraadams** [comment]
aha, i see why: https://github.com/no23reason/react-geolocated/blob/master/src/components/geolocated.js#L28

**Comment by no23reason** [comment]
Hi, this is a good point, configuring this should be possible. If you add an explanation into the README, I'm happy to merge the change you propose in your fork :)

**Comment by tyleraadams** [comment]
Awesome, just made pr.

**Comment by no23reason** [comment]
Fixed in #216 closing. The release should be ready soon :) thank you for contributing @tyleraadams !
